topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.md

```diff
@@ -283,23 +294,10 @@ For this tutorial, we will use the __SILVA database__ ({% cite Quast2012 %}). It
 {: .hands_on}
 
 
-> ### {% icon question %} Questions
->
-> What does the term k-mer mean?
-> 
->
-> > ### {% icon solution %} Solution
-> >
-> >  The term k-mer refers to all the sequence's subsequences of length _k_. For example, the sequence `GATACA` would have six monomers (`G`,`A`,`T`,`A`,`C`,`A`), five 2-mers (`GA`,`AT`,`TA`,`AC`,`CA`), four 3-mers (`GAT`,`ATA`,`TAC`,`ACA`), three 4-mer (`GATA`,`ATAC`,`TACA`) and two 5-mer (`GATAC`,`ATACA`).
-> >
-> {: .solution}
->
-{: .question}
-
 
 # Analyze taxonomic assigment
 
-Once we have assigned the corresponding taxa to each sequence, the next step is to properly visualize the data, for which we will use the {% tool [Krona pie chart](toolshed.g2.bx.psu.edu/repos/crs4/taxonomy_krona_chart/taxonomy_krona_chart/2.7.1) %} tool ({% cite Ondov_2011 %}). But before that, we need to adjust the format of the data output from {% tool [Kraken2](toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.0.8_beta+galaxy0) %}.
+Once we have assigned the corresponding taxa to each sequence, the next step is to properly visualize the data, for which we will use the __Krona pie chart__ tool ({% cite Ondov_2011 %}). But before that, we need to adjust the format of the data output from __Kraken2__.
```

Comment:
why this change?

Reply:
I consider that to include that information every time makes reading difficult, and in addition, it is not the style of most of the training. This information is included in the hands_on block.